src/common/api/swagger.yaml

```diff
@@ -1637,6 +1642,18 @@ paths:
           type: string
           enum: [discovery.ign, ipxe-script]
           required: true
+        - in: query
```

Comment:
I'd say the no-unrelated-changes rule should apply for deletions as well.

In general, we need to be sure that the changes we make in these files don't cause unwanted side effects that haven't been caught in manual testing because the task wasn't related to that part of the Installer. Perhaps we could do a separate task from time to time where we have a proper look at all of the possible changes in the swagger.yaml and types files.

Reply:
I executed "yarn update-api" to had my swagger updated. We need to think about @jgyselov says about make a proper look at all possible changes in swagger.yaml and types files.
I'll modify to have only the changes to swagger/types related with this task.